### DISCUSSION TITLE
How can i get Environment Variables?

### DISCUSSION THREAD

**Discussion by syinyang** [vercel/turborepo#3239] [Help] [answered]
--.env
--apps--docs--pages--index.ts
--turbo.json
--package.json

.env
DATABASE_URL="postgres://"

turbo.json
{
  "$schema": "https://turbo.build/schema.json",
  "pipeline": {
    "build": {
      "dependsOn": ["^build"],
      "outputs": ["dist/**", ".next/**"]
    },
    "lint": {
      "outputs": []
    },
    "dev": {
      "cache": false,
      "env": ["DATABASE_URL"]
    }
  }
}

package.json
{
"scripts": {
    "dev": "dotenv -- turbo run dev ",
  }
}

How can I get the DATABASE_URL ?

**Comment by anthonyshew** [comment, marked as answer]
Not sure if we'll be able to accurately answer this question with the information given here. I'm going to close it due to staleness but it looks like you may want to check out the `with-prisma` example: https://github.com/vercel/turbo/tree/main/examples/with-prisma
